### PR TITLE
Switch to GitHub Actions

### PR DESCRIPTION
**Pull request by marvinhagemeister**
See #2126 for more details.

### CODE REVIEW

File: .github/build-test.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Node CI
```
Comment (sventschui) [inline .github/build-test.yml]: You need to put this file into .github/workflows folder ;)

Comment (marvinhagemeister) [reply .github/build-test.yml]: Whoops! :sweat_smile:

File: .github/workflows/build-test.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          user: $SAUCE_USERNAME
+          key: $SAUCE_ACCESS_KEY
+      - name: npm install, build, and test
+        run: |
+          npm ci
+          npm run build --if-present
+          COVERAGE=true npm test
+          ./node_modules/coveralls/bin/coveralls.js < ./coverage/lcov.info;
+        env:
+          CI: true
```
Comment (developit) [inline .github/workflows/build-test.yml]: Adding [this](https://github.com/babel/preset-modules/blob/dff636d75ca8760a4882c7b65252fe443546c242/.github/workflows/main.yml#L26-L32) as the last action and removing the current SauceLabs setup should work:

```
    - name: saucelabs
      env:
        CI: true
        SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
        SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
      run: |
        npm run test:karma
```

We will just need to change the `test` script to skip karma tests in CI.

**Comment by developit** [comment]
I'm hoping Actions also gives us nicer ways to put file size on PRs. I looked into doing this a while back but it was more cumbersome than I'd expected.

**Pull request closed without merging**